.vscode/settings.json

```diff
@@ -2,17 +2,21 @@
     "json.schemas": [
         {
             "fileMatch": [
-                "./json/*/*.json"
+                "../json/000/*.json"
             ],
-            "url": "./schema.entry.json"
+            "url": "./schemas/entry.json"
         },
         {
             "fileMatch": [
                 "./list.json"
             ],
-            "url": "./schema.list.json"
+            "url": "./schemas/list.json"
         }
     ],
+    "yaml.schemas": {
+        "./schemas/entry.json": ["./yaml/*/*.yaml"],
+        "./schemas/list.json"
+    },
```

Comment:
Codacy found an issue: [Unexpected character ('}' (code 125)): was expecting a colon to separate field name and value](https://app.codacy.com/gh/ryuuganime/ryuuganime-db/pullRequest?prid=5948102)